src/aria/pageEngine/utils/BaseNavigationManager.js

```diff
@@ -55,9 +57,28 @@ module.exports = Aria.classDefinition({
          */
         this._navigate = cb || null;
 
+        var storageAvailable = true;
```

Comment:
I think doing it the opposite way would be better: initialize the variable with `false`, then set it to true after successfully assigning `this._storage`. This way, you can remove the `else` branch below.
And, maybe, you could even remove the `storageAvailable` variable completely and just check `this._storage` in the `if` statement.